packages/web-components/fast-ssr/src/template-parser/marker.ts

```diff
@@ -0,0 +1,19 @@
+import { DefaultTreeCommentNode } from "parse5";
+import { Markup } from "@microsoft/fast-element";
+
+const { marker } = Markup;
+const blockMarker = new RegExp(`${marker}:\\d+`);
+export function isMarkerComment(node: DefaultTreeCommentNode): boolean {
+    return blockMarker.test(node.data);
+}
+
+const interpolationMarker = new RegExp(`${marker}\\{(?<id>\\d+)\\}${marker}`);
+export function isInterpolationMarker(node: { value: string }): boolean {
+    return interpolationMarker.test(node.value);
+}
+
+export function extractInterpolationMarkerId(node: { value: string }): number | null {
+    const id = interpolationMarker.exec(node.value)?.groups?.id;
+
+    return id === undefined ? null : parseInt(id, 10);
+}
```

Comment:
For the above function, is there any way we could leverage what's in fast-element? If there were additional functions that took strings or something like that. Just trying to see if we can remove the `marker` dependency and not duplicate some of the pattern/parsing bits.

Reply:
Yeah actually, I think I might just be able to use the parser for all of this - let me try that.

Reply:
cool that all worked, I was able to remove this file